ent/encryption/encryption.cc

```diff
@@ -472,6 +472,14 @@ class encryption_context_impl : public encryption_context {
             for (auto&& [id, h] : _per_thread_kmip_host_cache[this_shard_id()]) {
                 co_await h->disconnect();
             }
+            static auto stop_all = [](auto&& cache) -> future<> {
+                for (auto& [k, host] : cache) {
```

Comment:
nit, 
C++26 will have a [nice placeholder with no name](https://github.com/cplusplus/papers/issues/878)
```suggestion
                for (auto& [_, host] : cache) {
```